bonsai-core/src/test/scala/com/stripe/bonsai/BitsetSpec.scala

```diff
@@ -18,7 +18,7 @@ class IndexedBitSetSpec extends WordSpec with Matchers with Checkers {
 
   implicit val arbitraryBitSetWithIndex: Arbitrary[BitSetWithIndex] =
     Arbitrary(for {
-      len <- arbitrary[Short].map(_.toInt)
+      len <- arbitrary[Short].map(_.toInt) if len >= 0
```

Comment:
`0xffff` is a positive `Int` (`65535`); that map changes the type as well as the value.